### PR TITLE
Display message when user search query is empty

### PR DESCRIPTION
**Pull request by conoroshea1996**
Fixed the issue when searching test it would break the site also added so now when the user searches and gets any empty result it no longer throws error and display no results found

### CODE REVIEW

File: src/components/Ticketmaster.js [hunk 1]
```diff
@@ -242,22 +245,29 @@ const Ticketmaster = function () {
...
         <tbody>
-          {data["_embedded"]["events"].map(tmevent => (
-            <tr key={tmevent.id}>
-              <td>{tmevent.name}</td>
-              <td>{createDateLocation(tmevent.dates.start.localDate, tmevent._embedded.venues[0].name)}</td>
-              <td>{createCityState(tmevent)}</td>
-              <td><a href={tmevent.url} rel="noopener noreferrer" target="_blank">Buy Tickets</a></td>
-            </tr>
-          ))}
+          {isResults ?
```
Comment (carlzoo) [inline src/components/Ticketmaster.js]: this would be better off stored in a variable

File: src/components/Ticketmaster.js [hunk 2]
```diff
@@ -242,22 +245,29 @@ const Ticketmaster = function () {
...
+            data["_embedded"]["events"].map(tmevent => (
```
Comment (carlzoo) [inline src/components/Ticketmaster.js]: this should be moved to a variable

File: src/components/Ticketmaster.js [hunk 3]
```diff
@@ -242,22 +245,29 @@ const Ticketmaster = function () {
...
+              <tr key={tmevent.id}>
+                <td>{tmevent.name}</td>
+                <td>{createDateLocation(tmevent.dates.start.localDate, tmevent)}</td>
+                <td>{createCityState(tmevent)}</td>
+                <td><a href={tmevent.url} rel="noopener noreferrer" target="_blank">Buy Tickets</a></td>
+              </tr>
+            )) :
+            <h1>No results found</h1>
```
Comment (carlzoo) [inline src/components/Ticketmaster.js]: should this be a modal? both?

Comment (conoroshea1996) [reply src/components/Ticketmaster.js]: I  think both would be nice I will look into having a modal aswell

**Pull request merged**